.github/workflows/ci-testing.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       matrix:
         platform: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: [3.6, 3.7, 3.8]
```

Comment:
question for @IAlibay : 2.0.0 will still support 3.6, right? At least that's the current state in setup.py https://github.com/MDAnalysis/mdanalysis/blob/d7b3c2481f43a5a1e39065058c52b1d0eb6672a0/package/setup.py#L575-L580

Then mdacli should mirror these requirements

Reply:
Okay. I thought 2.0 had dropped `3.6`.

Reply:
Sorry yeah my bad I should have explained better. 2.0.0 will still support py 3.6, that being said, py 3.6 is being dropped by a ton of places, so my earlier comment was more of a "since this is new, does it matter if it doesn't track directly with MDAnalysis core?"

Reply:
Hehe, okay. no problem. not a big deal.